map/style_tests/style_symbols_consistency_test.cpp

```diff
@@ -1,31 +1,30 @@
-#include "testing/testing.hpp"
 #include "helpers.hpp"
+#include "testing/testing.hpp"
```

Comment:
probably without blank clang-format resort consequential includes. ok, going to fix